packages/system/kubevirt-operator/Makefile

```diff
@@ -1,19 +1,12 @@
 NAME=kubevirt-operator
 NAMESPACE=cozy-kubevirt
 
-show:
-	helm template --dry-run=server -n $(NAMESPACE) $(NAME) .
-
-apply:
-	helm upgrade -i -n $(NAMESPACE) $(NAME) .
-
-diff:
-	helm diff upgrade --allow-unreleased --normalize-manifests -n $(NAMESPACE) $(NAME) .
+include ../../hack/package-system.mk
 
 update:
 	rm -rf templates
 	mkdir templates
 	export RELEASE=$$(curl https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirt/stable.txt) && \
-	wget https://github.com/kubevirt/kubevirt/releases/download/$${RELEASE}/kubevirt-operator.yaml -O templates/kubevirt-operator.yaml && \
-	sed -i 's/namespace: kubevirt/namespace: cozy-kubevirt/g' templates/kubevirt-operator.yaml
-	awk -i inplace -v RS="---" '!/kind: Namespace/{printf "%s", $$0 RS}' templates/kubevirt-operator.yaml
+	wget https://github.com/kubevirt/kubevirt/releases/download/$${RELEASE}/$(NAME).yaml -O templates/$(NAME).yaml && \
+	sed -i 's/namespace: kubevirt/namespace: $(NAMESPACE)/g' templates/$(NAME).yaml
+	awk -i inplace -v RS="---" '!/kind: Namespace/{printf "%s", $$0 RS}' templates/$(NAME).yaml
```

Comment:
I would rather keep update target without using variables, because commands here targets to concrete repositories, they are node depended on release name or namespace.

However I don't mind of replacing namespace with `$(NAMESPACE)` on line 11